tests/gh-actions/install_calico.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+kubectl create -f https://raw.githubusercontent.com/projectcalico/calico/v3.26.0/manifests/tigera-operator.yaml
+kubectl create -f https://raw.githubusercontent.com/projectcalico/calico/v3.26.0/manifests/custom-resources.yaml
+
+kubectl wait --for=condition=Ready pods --all --namespace=tigera-operator --timeout 300s
```

Comment:
This does not ensure that Calico is in a running state. This only ensures that the Tigera operator is.
Also: you should generally speaking not wait for Pods to be ready, but instead wait for the controlling Deployment, DaemonSet, StatefulSet, etc. to report that it is ready.

I got the following to work on my machine:

```suggestion
kubectl rollout status deployment -n tigera-operator tigera-operator --timeout 60s

# TODO: use 'kubectl wait --for=create namespace/calico-system' when https://github.com/kubernetes/kubernetes/pull/125868 is released
echo 'Waiting for namespace "calico-system" to be created'
start=$(date +%s)
while [ -z "$(kubectl get ns calico-system 2>/dev/null)" ]; do
  sleep 1
  now=$(date +%s)
  [ $((now - start)) -le 60 ] || { echo "timeout waiting for calico-system namespace" && exit 1; }
done

kubectl rollout status deployment -n calico-system calico-typha --timeout 60s
kubectl rollout status deployment -n calico-system calico-kube-controllers --timeout 60s

kubectl rollout status daemonset -n calico-system calico-node --timeout 60s
kubectl rollout status daemonset -n calico-system csi-node-driver --timeout 60s
```